packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -1601,24 +1601,29 @@ class BoxDecoration extends Decoration {
     );
   }
 
+  @protected
   @override
-  DiagnosticsNode toDiagnosticsNode({ String name, DiagnosticsTreeStyle style: DiagnosticsTreeStyle.whitespace }) {
```

Comment:
The meat of this CL is these refactors from using DiagnosticsNode.lazy to just extending Diagnosticable or DiagnosticableTree.